lib/patch-velero.sh

```
@@ -0,0 +1,76 @@
+# components/nordri/lib/patch-velero.sh
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)